translate.py

```diff
@@ -71,11 +73,15 @@ def addone(f):
 
 def main():
     opt = parser.parse_args()
+    dummy_parser = argparse.ArgumentParser(description='train.py')
+    add_model_arguments(dummy_parser)
+    dummy_opt = dummy_parser.parse_args()
```

Comment:
This line needs to be `dummy_opt = dummy_parser.parse_known_args()[0]`. Otherwise you'll get a parsing error when starting the script.